ontology/yaml/resources/HVAC/entity_types/HWS.yaml

```diff
@@ -325,4 +325,11 @@ HWS_SWTC_CPC:
   uses:
   - circulation_pump_speed_percentage_command
   - circulation_pump_speed_percentage_setpoint
-  - failed_alarm
\ No newline at end of file
+  - failed_alarm
+  
+HWS_ST:
```

Comment:
please make this HWS_SS, implement SS below, and remove the field usage. in fact, the type `HWS_SS_ALS` already exists so I suggest using that.

Reply:
I see. Will do. Thanks.